main.go

```diff
@@ -23,7 +23,10 @@ import (
 
 	"github.com/buchgr/bazel-remote/config"
 	"github.com/buchgr/bazel-remote/server"
+	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	httpmetrics "github.com/slok/go-http-metrics/metrics/prometheus"
+	httpmiddleware "github.com/slok/go-http-metrics/middleware"
```

Comment:
This is a relatively unknown library with only a single author. Could we use the promhttp import one line above instead?

Reply:
It's already using prometheus. This library is adding basic metrics for each HTTP endpoint.
See https://github.com/slok/go-http-metrics/blob/master/metrics/prometheus/prometheus.go
I just don't want to rewrite more code

Reply:
Oh, these are more useful metrics than `promhttp.Handler()` adds itself. OK.